docs/modules/ROOT/pages/how-to/customize-cypher.adoc

```diff
@@ -0,0 +1,302 @@
+= Customize Cypher
+
+In some cases, you may need to write highly customized queries within the Cypher Builder. For instance if you:
+
+* Need to embed Cypher strings into existing Cypher queries
+* Use Cypher Builder within a larger Cypher query
+* Need to use features that are not supported in the current version of the Cypher Builder
+* Use custom functions or procedures
+
+
+[WARNING]
+====
+Embedding custom Cypher in a query may lead to Code Injection and other security issues.  
+====
+
+== Custom Variable Names
+
+Usually, Cypher Builder takes care of variable names, ensuring these are unique and do not collide. In some cases, you may need to explicitly set names, to do it, you can use the `Named*` variables:
```

Comment:
In the linked page (https://neo4j.github.io/cypher-builder/cypher-builder/current/variables-and-params/#_named_variables) you can see that there are several elements called `Named` and something else, this refers to all of them. Didn't want to copy the entire list to avoid things falling out of sync and though on using `Named*` but, probably it is better to keep it simply as "named variables"